pkg/api/handlers/compat/containers_logs.go

```diff
@@ -106,6 +106,13 @@ func LogsFromContainer(w http.ResponseWriter, r *http.Request) {
 
 	w.WriteHeader(http.StatusOK)
 
+	flush := func() {
+		if flusher, ok := w.(http.Flusher); ok {
```

Comment:
Using https://go.dev/doc/go1.20#http_responsecontroller would be a bit simpler.

Reply:
I guess so, but this is pattern used in many endpoints so I guess we could do such change in one solid PR to switch all callers over

Reply:
Fair point, let’s defer that.